src/content/app/genome-browser/components/browser-track-config/track-config-views/RegularTrackConfig.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import SlideToggle from 'src/shared/components/slide-toggle/SlideToggle';
+
+import { useAppSelector, type RootState } from 'src/store';
+import {
+  getBrowserSelectedCog,
+  getTrackConfigsForTrackId
+} from 'src/content/app/genome-browser/state/track-config/trackConfigSelectors';
+
+import useBrowserTrackConfig from '../useBrowserTrackConfig';
+
+import styles from '../BrowserTrackConfig.scss';
+
+export const RegularTrackConfig = () => {
+  const selectedCog = useAppSelector(getBrowserSelectedCog) || '';
+  const selectedTrackConfigInfo = useAppSelector((state: RootState) =>
+    getTrackConfigsForTrackId(state, selectedCog)
+  );
+  const shouldShowTrackName = selectedTrackConfigInfo?.showTrackName ?? false;
+
+  const { updateTrackName } = useBrowserTrackConfig();
+
+  return (
+    <div className={styles.section}>
+      <div className={styles.subLabel}>Show</div>
+      <div>
+        <div className={styles.toggleWrapper}>
+          <label>Track name</label>
+          <SlideToggle
+            isOn={shouldShowTrackName}
+            onChange={() => updateTrackName(!shouldShowTrackName)}
```

Comment:
Use a function expression instead of an inline function?

Reply:
you mean have a separate function to call this? Sorry, I didnt get you.

Reply:
Write a function like this above and use it (`handleChange`) here:

```const handleChange = () => updateTrackName(!shouldShowTrackName)```;

Reply:
OK, thanks. So basically the main function called inside `handleChange`. Wondering the benefits of this as there are 4 functions in that component that would require this change.

Reply:
Let it be like this for now